spec/models/status_spec.rb

```diff
@@ -1,5 +1,24 @@
 require "rails_helper"
 
 RSpec.describe Status, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.